include/rdpq.h

```diff
@@ -952,19 +953,108 @@ inline void rdpq_set_blend_color(color_t color)
  * See #RDPQ_COMBINER1 and #RDPQ_COMBINER2 on how to configure
  * the color combiner (typicall, via #rdpq_mode_combiner).
  * 
+ * If you wish to set PRIM LOD or PRIM MIN LOD values of the PRIM register,
+ * see #rdpq_set_prim_lod, #rdpq_set_min_lod or #rdpq_set_prim_register_raw.
+ * 
  * @param[in] color             Color to set the PRIM register to
  * 
  * @see #RDPQ_COMBINER1
  * @see #RDPQ_COMBINER2
  * @see #rdpq_set_env_color
  * @see #rdpq_mode_combiner
+ * @see #rdpq_set_prim_lod
+ * @see #rdpq_set_min_lod
+ * @see #rdpq_set_prim_register_raw
  * 
  */
 inline void rdpq_set_prim_color(color_t color)
 {
     // NOTE: this does not require a pipe sync
     extern void __rdpq_write8(uint32_t cmd_id, uint32_t arg0, uint32_t arg1);
-    __rdpq_write8(RDPQ_CMD_SET_PRIM_COLOR, 0, color_to_packed32(color));
+    __rdpq_write8(RDPQ_CMD_SET_PRIM_COLOR_COMPONENT, (0<<16), color_to_packed32(color));
+}
+
+/**
+ * @brief Set the RDP MIN LOD combiner register (RDP command: SET_PRIM_COLOR (partial))
+ * 
+ * This function sets the internal RDP PRIM MIN LOD register, that is used for 
+ * determining the interpolation blend factor of a detail texture.
```

Comment:
Should we maybe rename it in another way? I don't know the story behind "MIN LOD", but if we use it as blend factor for detail texture, maybe it should be `rdpq_set_blend_detail_factor()` or something like that.